packages/next-codemod/transforms/lib/async-request-api/next-async-dynamic-prop.ts

```diff
@@ -22,6 +22,7 @@ import {
 } from './utils'
 
 const PAGE_PROPS = 'props'
+const MATCHED_FILE_PATTERNS = /([\\/]|^)(page|layout|route)\.(t|j)sx?$/
```

Comment:
Could there be edge case with `nextConfig.pageExtensions`?

Reply:
that's able to pass down to codemod, to keep codemod simple just matching all of them. This line was just moving around, unrelated to this PR